howto/troubleshoot/landing.md

```diff
@@ -1,11 +1,31 @@
 The topics in this guide describe some commonly encountered problems with Anbox Cloud and provide instructions for resolving them. If you encounter an issue with Anbox Cloud, check if any of the following scenarios help in resolving your issue.
 
+[note type="information" status="Note"]If the deployment is older than 3 months, you must upgrade Anbox Cloud to the latest version and see if the required fixes are already part of the upgrade. See [How to upgrade Anbox Cloud](https://discourse.ubuntu.com/t/how-to-upgrade-anbox-cloud/17750) for upgrade instructions.
+[/note]
+
 * [Troubleshoot issues with initial setup](https://discourse.ubuntu.com/t/troubleshoot-issues-with-initial-setup/35704)
 * [Troubleshoot container failures](https://discourse.ubuntu.com/t/troubleshoot-container-failures/35703)
 * [Troubleshoot issues with application creation](https://discourse.ubuntu.com/t/troubleshoot-issues-with-application-creation/35702)
 * [Troubleshoot issues with LXD clustering](https://discourse.ubuntu.com/t/troubleshoot-issues-with-lxd-clustering/35705)
 
-If you still need help, use the following instructions to collect troubleshooting information from the container and report an issue.
+If you still need help, use any of the following utilities to collect troubleshooting information and report an [issue](https://bugs.launchpad.net/anbox-cloud/+filebug).
```

Comment:
So... As a user, what should I do when I see these? Run all three? Run only the relevant ones? How should I know which ones are relevant?

I think we need to provide some guidelines for users. What do you think?

Reply:
I think it depends on what questions support/we ask them. Let me check into this further.

Reply:
@morphis When you/support have asked for log files or crashdump in the past, what of these three options did you ask? The Juju crashdump (provided the customer has the plugin installed), logs from Anbox report utility, stored container logs. 

We are looking to provide some context on which one to run before filing a bug or will doing all of this help?

Reply:
It depends on where the actual issue is. A crashdump gives you the high level overview, so is definitely a good way to start (or with the appliance `anbox-cloud-appliance.buginfo`).  The other two provide more low level information which are needed in some cases. If someone files a bug the start should always be a crashdump (for regular) or `anbox-cloud-appliance.buginfo` (for appliance): The other two can then be asked for as follow up information.